doc/api/http2.md

```diff
@@ -1899,7 +1899,9 @@ const req = client.request({
   ':authority': `localhost:${port}`
 });
 
-req.on('response', common.mustCall());
+req.on('response', (headers) => {
+  console.log(headers[':status']);
```

Comment:
Would be great if this is `headers[http2.constants.HTTP2_HEADER_STATUS]`

Reply:
Hey, thanks for the review. I just made the necessary changes to the line.